src/Player.ts

```
@@ -92,6 +92,17 @@ export class Player {
       return 0;
     }
 
+    public getResource(resource: Resources): number {
+      if (resource === Resources.MEGACREDITS) return this.megaCredits;
+      if (resource === Resources.STEEL) return this.steel;
+      if (resource === Resources.TITANIUM) return this.titanium;
+      if (resource === Resources.PLANTS) return this.plants;
+      if (resource === Resources.ENERGY) return this.energy;
+      if (resource === Resources.HEAT) return this.heat;
+      return 0;
```

Instead of returning `0` we should throw an error here. This branch represents a `Resources` we are not considering. If some new resource is added at a later time it could fail fast to remind us to add the enum here.